src/Symfony/Component/Process/Process.php

```diff
@@ -334,6 +334,14 @@ public function start(callable $callback = null/*, array $env = array()*/)
             $ptsWorkaround = fopen(__FILE__, 'r');
         }
 
+        if (!is_dir($this->cwd)) {
+            if ('\\' === DIRECTORY_SEPARATOR) {
+                throw new RuntimeException('The provided cwd does not exist.');
+            }
+
+            @trigger_error('The provided cwd does not exist. Command is currently ran against getcwd(). This behaviour is deprecated since version 3.4 and will be removed in 4.0.', E_USER_DEPRECATED);
```

Comment:
behavior

Reply:
Does Symfony follow british or american spelling across the board?

Reply:
US

Reply:
Ok, ill make that change this evening

Reply:
Done